PYTHON/Lakehouse/query_data_from_tables.py

```diff
@@ -0,0 +1,34 @@
+# This sample reads data from a table in a lakehouse 
+# Complete these steps before testing this function
+# 1. Select Manage connections and add a connection to a Lakehouse 
+
+import datetime
+import json
+
+@app.fabric_item_input(argName="mylakehouse", alias="<My Lakehouse alias>")
```

Comment:
some samples currently use app.connection, and others use fabric_item_input, so we should make them consistent when we decide which to use